extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -59,9 +61,10 @@ object KinesisUtils {
       endpointUrl: String,
       checkpointInterval: Duration,
       initialPositionInStream: InitialPositionInStream,
-      storageLevel: StorageLevel): ReceiverInputDStream[Array[Byte]] = {
+      storageLevel: StorageLevel,
+      credentials: Option[AWSCredentials]): ReceiverInputDStream[Array[Byte]] = {
```

Comment:
updated the parameter with default value ```None``` which means ```DefaultAWSCredentialsProvider``` will be used in the end.

However i couldn't to do the same for the Java api. Any suggestions?